amd/src/personaltutor.js

```diff
@@ -40,41 +40,42 @@ define([
                 $('#tutortable').DataTable({
                     'dom': 'lfBrtip',
                     fixedHeader: true,
-                    'order': [1, 'asc' ],
+                    'order': [1, 'asc'],
                     'columnDefs': [
-                        {'orderable': false, 'targets': 0 }],
-                    buttons: [ 'colvis' ],
+                        {'orderable': false, 'targets': 0}],
+                    buttons: ['colvis'],
                     responsive: true
                 });
 
-                $('#selectall').change(function(){
+                $('#selectall').change(function() {
                     $('.chk1').prop('checked', $(this).prop('checked'));
                 });
                 $("#emailform").click(function() {
                     var mylink = [];
-                    $("input:checked").each(function(){
+                    $("input:checked").each(function() {
                         if ($(this).val() != 'on') {
                             mylink.push($(this).val());
                         }
                     });
                     if (mylink.length > 0) {
-                        str.get_string('email_tutee_subject', 'report_myfeedback').then((email_tutee_subject) => {
-                            $("a#mail").attr("href", "mailto:?bcc=" + mylink.join(";") + "&Subject=" + email_tutee_subject + "");
-                        });
+                        // eslint-disable-next-line promise/always-return
```

Comment:
I'm assuming you've reviewed https://github.com/eslint-community/eslint-plugin-promise/blob/main/docs/rules/always-return.md and there's no way to make it comply?

Reply:
I now have addressed this properly.